packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -117,25 +133,276 @@ export default class VTable {
      * @param format Table format to apply
      */
     applyFormat(format: Partial<TableFormat>) {
-        if (!format || !this.table) {
+        if (!this.table) {
             return;
         }
+        this.formatInfo = {
+            ...(this.formatInfo || {}),

```

Comment:
What if this.formatInfo is null and format only contains part of the properties?

You need to apply default format first.